homeassistant/components/cast/manifest.json

```diff
@@ -4,7 +4,8 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/cast",
   "requirements": ["pychromecast==7.2.1"],
-  "after_dependencies": ["cloud", "http", "media_source", "tts", "zeroconf"],
+  "dependencies": ["zeroconf"],
```

Comment:
We shouldn't change this. `after_dependencies` means that `cast` is set up after that `zeroconf` is set up if `zeroconf` is requested by the user. With this change, enabling `cast` will also enable `zeroconf` to discover devices on the network.

Reply:
I wonder if there is a use case were we want to enable `zeroconf` so we can set `default_interface` or disable `ipv6`, but do not want discovery.

I guess if nobody has asked, its not worth implementing.

I'll rework this to address the `after_dependencies` issue

Reply:
done